docs/changelog.md

```diff
@@ -2,6 +2,97 @@
 
 Beanie project
 
+## [1.27.0] - 2024-10-06
```

Comment:
Only one slight "issue" now with the approval coming late is that yesterdays date is for the 1.27.0 "release".
Also, we should also release the documentation, are there efforts to fix the docs publish action?

Reply:
If we were to update the date, it would automatically dismiss the previous approvals, and require everyone to approve again. I think it's best if we just get this release out of the way and move on to the next release.

Reply:
Agree. Just wanted to point this out.